code/Controllers/LeftAndMainPageIconsExtension.php

```diff
@@ -2,32 +2,58 @@
 
 namespace SilverStripe\CMS\Controllers;
 
+use Psr\SimpleCache\CacheInterface;
+use Psr\SimpleCache\InvalidArgumentException;
+use ReflectionException;
 use SilverStripe\CMS\Model\SiteTree;
 use SilverStripe\Core\ClassInfo;
-use SilverStripe\Core\Config\Config;
```

Comment:
Removing this use statements seems to be breaking the build.

Reply:
Weird, was showing as unused in my IDE so I blindly removed it, have blindly added it back in